templates/cassiopeia/scss/blocks/_header.scss

```diff
@@ -155,7 +179,7 @@
       margin-right: 12px;
       margin-left: 5px;
       font-size: $h1-font-size;
-      color: lighten($cassiopeia-template-color, 10%);
+//      color: scale-color(var(--cassiopeia-color-primary), $lightness: 10%);
```

Comment:
```suggestion
```
Shall it be removed, or is it some work in progress? If to be removed, just apply this suggestion, otherwise let me know.

Reply:
The problem is that scale-color with a css variable doesn't compile, because the color is not present. We will probably need some pre-defined color here, but I'm nor sure for what the color is being used.

Reply:
@hans2103 Do you have any idea?

Reply:
Solved with commit https://github.com/joomla/cassiopeia/pull/71/commits/4ae6c300931602d7223556d79f247317b6e990a1 . Thanks @hans2103

Reply:
Ah, no sorry, wrong one. Is not solved yet.

Reply:
My suggestion would be removing the entire block `.container-title`. I cannot find it being used in Joomla frontend.